gateway/mdb/views/mdbping.py

```diff
@@ -0,0 +1,26 @@
+from django.http import (
+    HttpRequest,
+    HttpResponse,
+    HttpResponseBadRequest,
+)
+from django.utils import timezone
+
+from ..models import Machine
+
+
+def mdbping(request: HttpRequest) -> HttpResponse:
```

Comment:
When using an API route with POST request, you should use the csrf_exempt decorator